test-fixture/src/lib.rs

```diff
@@ -167,7 +166,23 @@ pub fn new_client(params: ConnectionParameters) -> Connection {
         now(),
     )
     .expect("create a client");
-    client.set_qlog(log);
+
+    if let Ok(dir) = std::env::var("QLOGDIR") {
```

Comment:
Should document this `env` variable somewhere.

Reply:
:+1: see https://github.com/mozilla/neqo/pull/1868/commits/875b93ee8d6fce2a80548888832757f60254e675.